tests/5.0/loop/test_loop_reduction_min.F90

```diff
@@ -0,0 +1,85 @@
+!===--- test_loop_reduction_min.F90 -------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test uses the reduction clause on a loop directive, testing that the
+! variable in the reduction clause is properly reduced using the min
+! operator.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 1024
+
+PROGRAM test_loop_reduction_min
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_VERBOSE(test_min() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+  INTEGER FUNCTION test_min()
+    INTEGER,DIMENSION(N):: a, b, num_threads
+    INTEGER:: errors, device_result, host_min, x
+    CHARACTER(len=400) :: msgHelper
+    INTEGER :: seedSize
+    INTEGER,ALLOCATABLE :: seed(:)
+    DOUBLE PRECISION:: randomNumber
+    INTEGER:: randomInteger
+
+    errors = 0
+    CALL random_seed(size=seedSize)
+    ALLOCATE(seed(seedSize))
+    seed = 1
+    CALL random_seed(put=seed)
+    DEALLOCATE(seed)
+
+    DO x = 1, N
+       !random_number() generates a real number, r, uniformly distributed in 0 <= r < 1.
+       CALL random_number(randomNumber)
+       randomInteger = INT(randomNumber*100.0)
+       a(x) = randomInteger
+       CALL random_number(randomNumber)
+       randomInteger = INT(randomNumber*100.0)
+       b(x) = randomInteger
+       num_threads(x) = -1 * x
+    END DO
+
+    device_result = a(1) + b(1)
+
+    !$omp parallel num_threads(OMPVV_NUM_THREADS_HOST)
+    !$omp loop reduction(min:device_result)
+    DO x = 1, N
+      device_result = MIN(a(x) + b(x), device_result)
```

Comment:
Is this intended for device execution ? If yes, then it is missing "target" at ln 55.

Reply:
I see a separate test. The "device_result" variable confused me. :)